form/form_dependencies.rst

```diff
@@ -71,36 +71,41 @@ Alternatively, you can define your form class as a service. This is a good idea
 you want to re-use the form in several places - registering it as a service makes
 this easier.
 
-Suppose you need to access the ``doctrine.orm.entity_manager`` service so that you
+Suppose you need to access the ``EntityManager`` service so that you
```

Comment:
All this is still confusing to me but ... is this statement correct? --> `[...] the EntityManager service [...]`

If I execute `./bin/console debug:container --types` I see this:

```
 --------------------------------------------  ------------------------------------------------
  Service ID                                   Class name
 --------------------------------------------  ------------------------------------------------
  Doctrine\Common\Annotations\Reader           alias for "annotations.cached_reader"
  Doctrine\Common\Persistence\ManagerRegistry  alias for "doctrine"
  Doctrine\Common\Persistence\ObjectManager    alias for "doctrine.orm.default_entity_manager"
  Doctrine\DBAL\Driver\Connection              alias for "doctrine.dbal.default_connection"
```

So there's no *"EntityManager service"*, right?